### ISSUE TITLE
A11y_AzureDataStudio_Selectthedeploymentoption_AI4D:Name property does not exist for I accept check box 

### ISSUE THREAD

**Issue by V-sheman** [V-sheman/Manisha#1]
"Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
GitHubTags: #A11y_AzureDataStudio;#A11yMAS;#A11yTCS;#MAS1.3.1,#MAS4.1.2;#MAS4.2.1;#DesktopApp;#electron.0,#NVDA
Environment Details:
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 32df727ff9fe68421b004bf270cd509e4c5de165
Date: 2019-10-08T19:18:43.743Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
Additional Details:
AI4D
**MAS References:**MAS1.3.1 MAS4.1.2, MAS4.2.1
**Steps to Reproduce :**
1.Install /Launch the application Azure data studio.
2.Navigate to connections and click on more action button.
3.Select “Deploy SQL Server big data cluster” from more action drop down.
4.Click on “Deploy SQL Server big data cluster” .
5.Open AI4D and hover on the I accept check box.
**Actual:**
When we hover on the check box we found name property does not exist for the check box and also it is not announced by the NVDA.
**Expected:
Name property should exist for the check box.
User Impact:
Screen reader User will not able to understand for what purpose the check box is there.
MAS Reference:
MAS1.3.1 -
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
MAS4.1.2 -
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}
MAS4.2.1 -
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a12f3779-bdf1-40c2-bbcc-0e314ea94858}

**Attachment for Reference:**
![image](https://user-images.githubusercontent.com/56344899/66474045-28ba0700-eaae-11e9-88c5-c7a1b10ff76c.png)
